src/NServiceBus.RabbitMQ.AcceptanceTests/DelayedDelivery/When_Deferring_a_message_longer_than_allowed_maximum.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace NServiceBus.AcceptanceTests.DelayedDelivery
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    public class When_Deferring_a_message_longer_than_allowed_maximum : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public void Should_throw()
+        {
+            var delay = TimeSpan.FromDays(365 * 10);
```

Comment:
Currently we're going with 28 bits which gives us a max 8 years, if we did decide to go with 31 bits (the max unless we switch from `int` to `long`) then anything above 64 years is enough to future proof this test.